scss/detail.scss

```diff
@@ -22,12 +20,25 @@
   display: flex;
   flex-direction: column;
   height: 368px;
-  margin-right: 52px;
-  overflow: hidden;
+  margin: 0 52px 0 0;
+  position: relative;
```

Comment:
I dont understand why no use margin-right: 52px?;

Reply:
I already changed it, thanks